Index: include/wx/msw/textctrl.h
===================================================================
--- include/wx/msw/textctrl.h	(revision 72949)
+++ include/wx/msw/textctrl.h	(working copy)
@@ -31,7 +31,7 @@
 
         Create(parent, id, value, pos, size, style, validator, name);
     }
-    virtual ~wxTextCtrl();
+    virtual ~wxTextCtrl() wxNOEXCEPT;
 
     bool Create(wxWindow *parent, wxWindowID id,
                 const wxString& value = wxEmptyString,
Index: include/wx/stdstream.h
===================================================================
--- include/wx/stdstream.h	(revision 72949)
+++ include/wx/stdstream.h	(working copy)
@@ -28,7 +28,7 @@
 {
 public:
     wxStdInputStreamBuffer(wxInputStream& stream);
-    virtual ~wxStdInputStreamBuffer() { }
+    virtual ~wxStdInputStreamBuffer() wxNOEXCEPT { }
 
 protected:
     virtual std::streambuf *setbuf(char *s, std::streamsize n);
@@ -71,7 +71,7 @@
 {
 public:
     wxStdInputStream(wxInputStream& stream);
-    virtual ~wxStdInputStream() { }
+    virtual ~wxStdInputStream() wxNOEXCEPT { }
 
 protected:
     wxStdInputStreamBuffer m_streamBuffer;
@@ -85,7 +85,7 @@
 {
 public:
     wxStdOutputStreamBuffer(wxOutputStream& stream);
-    virtual ~wxStdOutputStreamBuffer() { }
+    virtual ~wxStdOutputStreamBuffer() wxNOEXCEPT { }
 
 protected:
     virtual std::streambuf *setbuf(char *s, std::streamsize n);
@@ -112,7 +112,7 @@
 {
 public:
     wxStdOutputStream(wxOutputStream& stream);
-    virtual ~wxStdOutputStream() { }
+    virtual ~wxStdOutputStream() wxNOEXCEPT { }
 
 protected:
     wxStdOutputStreamBuffer m_streamBuffer;
Index: include/wx/textctrl.h
===================================================================
--- include/wx/textctrl.h	(revision 72949)
+++ include/wx/textctrl.h	(working copy)
@@ -679,7 +679,7 @@
     // --------
 
     wxTextCtrlBase() { }
-    virtual ~wxTextCtrlBase() { }
+    virtual ~wxTextCtrlBase() wxNOEXCEPT { }
 
 
     // more readable flag testing methods
Index: src/common/combocmn.cpp
===================================================================
--- src/common/combocmn.cpp	(revision 72949)
+++ src/common/combocmn.cpp	(working copy)
@@ -940,7 +940,7 @@
 {
 public:
     wxComboCtrlTextCtrl() : wxTextCtrl() { }
-    virtual ~wxComboCtrlTextCtrl() { }
+    virtual ~wxComboCtrlTextCtrl() wxNOEXCEPT { }
 
     virtual wxWindow *GetMainWindowOfCompositeControl()
     {
Index: src/generic/spinctlg.cpp
===================================================================
--- src/generic/spinctlg.cpp	(revision 72949)
+++ src/generic/spinctlg.cpp	(working copy)
@@ -77,7 +77,7 @@
         SetSizeHints(wxDefaultCoord, wxDefaultCoord);
     }
 
-    virtual ~wxSpinCtrlTextGeneric()
+    virtual ~wxSpinCtrlTextGeneric() wxNOEXCEPT
     {
         // MSW sends extra kill focus event on destroy
         if (m_spin)
